public/javascripts/payments.js

```diff
@@ -768,6 +858,10 @@
     if (paymentMethod === 'sepa_debit' && bankName) {
       label = `Debit ${amount} from ${bankName}`;
     }
+    if (paymentMethod == 'au_becs_debit' && bankName){
+      label = `Debit ${amount} from ${bankName}`;
+    }
```

Comment:
This looks the same as the `sepa_debit` case, so you could just expand that, instead:

```javascript
if (['sepa_debit', 'au_becs_debit'].include?(paymentMethod) && bankName) {
  label = `Debit ${amount} from ${bankName}`;
}
```

Reply:
Good suggestion! Made the udpate